### ISSUE TITLE
[Feature request] Copy rows as sql update

### ISSUE THREAD

**Issue by bmichotte** [TablePlus/TablePlus#951]
On the rows listing, we have a contextual menu which allows us to copy selected rows as Sql insert statement. Could you please add the same but as `update` ?

It would allows to quickly update rows from dev to stable database.

**Comment by whobutsb** [comment]
I was just looking for this feature.  I needed to copy all of the result IDs to do a DELETE WHERE IN statement.  This would be very helpful.

**Comment by huyphams** [comment]
You can do this with copy column feature @whobutsb
<img width="326" alt="Screen Shot 2019-04-03 at 11 28 38 PM" src="https://user-images.githubusercontent.com/3654622/55495991-43777300-5668-11e9-8446-5778b228871b.png">

**Comment by whobutsb** [comment]
Boom!  That's handy!  Copy Columns wasn't that clear, I expected it to copy the column names.  Now I know. Thanks!

**Comment by wooshifu** [comment]
Yep, I also find it very useful when we want to update some row data.
following pic is from [Navicat](https://www.navicat.com/en) implementation.
![image](https://user-images.githubusercontent.com/8529077/67734921-67076e00-fa3d-11e9-8358-c814f6db16e5.png)
![image](https://user-images.githubusercontent.com/8529077/67734870-3de6dd80-fa3d-11e9-87b0-8a6bec448ee1.png)

**Comment by kkor** [comment]
> You can do this with `Copy Columns` feature @whobutsb

@huyphams This doesn't seem to work anymore, it's now called "Copy all column values" and only copies the values as name says... is the update statement somewhere else or has it been removed completely?

**Comment by fhriz** [comment]
hi @huyphams, any news if this new feature request will be included on the next future release?

Currently when I need it, I need to create another script to translate the result of "Copy Rows As JSON" data to the correct SQL update query. It would be great if I can just Copy Rows As SQL Update statement.

Thanks for the great app!

**Comment by skw1335** [comment]
We had a great feature and then we removed it? Why :(
